WaysofWork.md

```diff
@@ -32,7 +32,10 @@ When created in another repository try to still apply these rules as much as pos
     - Backlog: if it has been agreed as necessary work
     - Planned: agreed and we know when to address it or by when it needs done 
   - Points (effort): assign a value effort from 2 to 9. A value of 1 typically means that the task is so short it would be better merged with another, a 10 means that it is probably best split in more than one issue (maybe it needs to be a story instead of an issue); if you really want to assign a 1 or 10 (cannot be merged or split) do so
+  - FTE: estimate % of person time while ongoing this issue will take, for example 2 people dedicating 1 day a week, wiht an issue duration of a month would be about 0.26.
+    - Important! Update this when closing
```

Comment:
Do we want to make this mandatory? there is interest on being able to analyse this

Reply:
This is true, also ties with 

> 38 - Add actual dates when closing

Let's only update/fill start date when actually starting (if issue was planned in advance). Date difference will give us what we need to analyse along with knowing who worked on them.

Effort: we are not realistically going to use FTE AND Points, so let's keep one or the other:
- FTE: we are more familiar with it and use it elsewhere BUT I dislike the pretense of actually knowing with proper detail what the task will take
- Points: are quick and provide a sense of the level of work, the resulting escale is well adjusted to the team after a time BUT is new, requires being persistant and consistent; It can be a bit of a fuss

While I prefer points I acknowledge FTE may work better for the team

Reply:
+1 for capturing actual effort spent if that's possible.

Reply:
I prefer estimating using FTE. I know it isn't accurate, but having the unit makes it clear what 1.0 FTE means. It is the same for everyone. I'm not sure how 2 of my points compare to 5 of yours.

It makes it easier to compare across people/projects and gives a better impression of how reasonable the time investment is, even if it is a rough estimate. @jemrobinson thoughts?

Reply:
@jemrobinson would you capture actual effort at a task or story level? there are two main ways I see of capturing it:
- Add a field for actual effort (instead of overwrite) and ask for it to be filled in when closing
- Ask for a closing comment with the effort and final outcomes, something of a mini report on closure. If this does not sit well with issues closed in other repositories, it could also be an update in the story "closed related issue ##, final effort X, outcome Y"
- The above but reporting in the weekly meeting/note

Any other suggestions?

Reply:
@JimMadge ok to using FTE, points do work and in the end the whole team comes to understand them but it requires time and probably following SCRUM more closely than we are doing. We are experimenting enough already

Reply:
Again think this is cool IF we're going to (and see a benefit from) analysing it